lib/modules/manager/bitrise/extract.spec.ts

```diff
@@ -0,0 +1,138 @@
+import { codeBlock } from 'common-tags';
+import { BitriseDatasource } from '../../datasource/bitrise';
+import { extractPackageFile } from '.';
+
+describe('modules/manager/bitrise/extract', () => {
+  describe('extractPackageFile()', () => {
+    it('returns null on an empty file', () => {
+      expect(extractPackageFile('', 'bitrise.yml')).toBeNull();
+    });
+
+    it('returns valid file', () => {
+      expect(
+        extractPackageFile(
+          codeBlock`
+      workflows:
+        test:
+          steps:
+          - script@1.1.5:
+      `,
+          'bitrise.yml',
+        ),
+      ).toEqual({
+        deps: [
+          {
+            datasource: BitriseDatasource.id,
+            packageName: 'script',
+            currentValue: '1.1.5',
+            replaceString: 'script@1.1.5',
+          },
+        ],
+      });
+    });
+
+    it('returns valid file with custom default_step_lib_source', () => {
+      expect(
+        extractPackageFile(
+          codeBlock`
+      format_version: 11
+      default_step_lib_source: https://github.com/custom/steplib.git
+      project_type: android
+      app:
+        envs:
+        - MY_NAME: My Name
+      workflows:
+        test:
+          steps:
+          - script@1.1.5:
+              inputs:
+              - content: echo "Hello!"
+          - restore-cache@1.1.2:
+              foo: bar
+      `,
+          'bitrise.yml',
+        ),
+      ).toEqual({
+        deps: [
+          {
+            datasource: BitriseDatasource.id,
+            packageName: 'script',
+            currentValue: '1.1.5',
+            replaceString: 'script@1.1.5',
+            registryUrls: ['https://github.com/custom/steplib.git'],
+          },
+          {
+            datasource: BitriseDatasource.id,
+            packageName: 'restore-cache',
+            currentValue: '1.1.2',
+            replaceString: 'restore-cache@1.1.2',
+            registryUrls: ['https://github.com/custom/steplib.git'],
+          },
+        ],
+      });
+    });
+
+    it('extract git and path prefixes', () => {
```

Comment:
```suggestion
    it('extracts Git and path prefixes', () => {
```

Reply:
the prefixes here `git::` and `path::` therefore I prefer to keep it this way